enterprise_subsidy/apps/api_client/enterprise_catalog.py

```diff
@@ -16,7 +16,7 @@ class EnterpriseCatalogApiClient(BaseOAuthClient):
     """
     API client for calls to the enterprise service.
     """
-    api_base_url = urljoin(settings.ENTERPRISE_CATALOG_URL, 'api/v1/')
+    api_base_url = urljoin(settings.ENTERPRISE_CATALOG_URL, 'api/v2/')
```

Comment:
Do we need every v1 endpoint in enterprise-catalog to have a functional v2 version (even if using the same exact logic as v1)?

Reply:
Nope, I was thinking of elaborating on this by adding a constructor argument to ask for the API version.